cloudbuild_public.yaml

```diff
@@ -0,0 +1,70 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: 'bash'
+  args: [
+    '-c',
+    'docker pull us.gcr.io/$PROJECT_ID/$REPO_NAME:nightly || true',
+  ]
```

Comment:
Strongly recommend using Kaniko as the builder to avoid this manual cache management: https://github.com/getsentry/sentry/blob/e087a7e02312cb85fc9b1bb758138e83e0a4559c/docker/cloudbuild.yaml#L2

Reply:
Added Kaniko builder